test/controllers/api/v2/products_controller_test.rb

```diff
@@ -65,6 +65,20 @@ def test_index_protected
       end
     end
 
+    def test_index_custom_products_only
+      get :index, params: { organization_id: @organization.id, custom: true }
+      body = JSON.parse(response.body)
+
+      assert_equal body['total'], 4
+    end
+
+    def test_index_no_custom_products
+      get :index, params: { organization_id: @organization.id, custom: false }
+      body = JSON.parse(response.body)
+
+      assert_equal body['total'], 3
```

Comment:
I don't know if there is a good way to not hardcode the counts here without duplicating the logic in the products controller